features/ui/checkbox/checkbox.tsx

```diff
@@ -12,15 +13,13 @@ export enum CheckboxState {
   indeterminate = "indeterminate",
 }
 
-interface CheckboxProps {
+export type CheckboxProps = InputHTMLAttributes<HTMLInputElement> & {
   label: string;
-  name: string;
   checkboxState: CheckboxState;
-  disabled: boolean;
-  size: CheckboxSize;
-}
+  checkboxSize: CheckboxSize;
```

Comment:
Imo you could write `size` and `state` here and drop the `checkbox` prefix on the props. It's clear that this is the checkbox size anyways. I think you did that in the rest of your code as well.

Reply:
I did attempt this originally, but realised the HTML Input props already include a `size` prop, so clashing with that causes errors. Not sure where my comment regarding this got lost to, but I'll add it back in. To avoid the clash, I added the checkbox prefix to both for consistency within this component at least.